integration/testUtils/usersService.ts

```diff
@@ -67,7 +67,7 @@ export const createUserService = (clerkClient: ClerkClient) => {
         withUsername = false,
       } = options || {};
       const randomHash = hash();
-      const email = fictionalEmail ? `${randomHash}+clerk_test@example.com` : `clerkcookie+${randomHash}@mailsac.com`;
+      const email = fictionalEmail ? `${randomHash}+clerk_test@clerkcook.ie` : `clerkcookie+${randomHash}@mailsac.com`;
```

Comment:
Wouldn't `clerkcookie.com` also work? Then you could leave the `query` in the `cleanup.setup.ts` the same

Reply:
We also use email as password and there is a limit on characteres so I tried to keep it short

Reply:
How about changing the password to `{email}{randomHash}` instead, then you could use the full `clerkcookie` name and wouldn't need to change the individual test. Better to fix the root cause than to fix it in an individual test. This way no one else will run into it.

What do you think? :)

Reply:
Agree with Lennart here :)

Reply:
The hash will be inside the email name so I think you can still figure out the password 👍